### ISSUE TITLE
BottomSheetState non-responsive when going back from SettingsScreen

### ISSUE THREAD

**Issue by simonalveteg** [simonalveteg/Minute-Launcher#13]
.expand() function does absolutely nothing, despite all other uiEvents working. In the example below both logs get logged
```
            Log.d("MAIN_SCREEN","expand sheet $bottomSheetScaffoldState")
            bottomSheetScaffoldState.bottomSheetState.expand()
            Log.d("MAIN_SCREEN","sheet expanded $bottomSheetScaffoldState")
```
Can't figure out the source of the issue, seems to be the scaffold state not responding?

**Comment by a1vv** [comment]
Using the default peek height of the bottom sheet it is possible to expand it by dragging the visible part, so the bottom sheet is part of the composition.

**Comment by simonalveteg** [comment]
Solved in [commit 678b1a8](https://github.com/simonalveteg/Minute-Launcher/commit/678b1a84b1faadc433bf89dd4378b370626c4b35). Used BottomSheetState instead of rememberBottomSheetState..
